### DISCUSSION TITLE
DatePicker - how to let user select day from other month?

### DISCUSSION THREAD

**Discussion by ZnowuJa** [microsoft/fluentui-blazor#3113] [General]
Hi,

I struggle with a limitation that it is not possible to select other than this months day  in FluentDatePicker.
For example this simple control:

```
    <FluentDatePicker Label="@Label" 
                      Value="@SelectedDate.ToDateTimeNullable()"
                      ValueChanged="@(e => OnDateChanged(e.ToDateOnlyNullable()))"
                      Style="width: 150px"
    />
```

 
 that looks like this:
 
![image](https://github.com/user-attachments/assets/9f26a2d9-7a55-4074-910a-fb02fdd42ad2)

All days from January are "unaccessible" and user has to click arrow down to let them.
I read all documentation and github repos discussions, but didn't find an answer.

If we could scroll months instead of pressing arrows up and down that would be great.

**Comment by dvoituron** [comment]
In fact, the only way to get to January is to click on the arrows.
You can of course use the numbers to choose a date.

This is how it works now and there are no plans to change it in the near future.

**Reply by ZnowuJa** [reply]
ok. Thank you for quick reply!
:)
